readme.md

```diff
@@ -1,7 +1,6 @@
-# Official Firefly III k8n files
+## Firefly III helm registry
```

Comment:
Moved to a level 2 heading as GitHub pages uses the repository name as level 1 heading, can be seen on https://firefly-iii.github.io/kubernetes/